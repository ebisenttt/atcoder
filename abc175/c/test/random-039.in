525083715938742 989928427973701 237912876834503
