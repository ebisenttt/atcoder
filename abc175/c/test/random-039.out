188654914564767
